iib/web/static/api_v1.yaml

```diff
@@ -648,6 +655,13 @@ components:
             index images that IIB does not have access to. The format of the token is in the format
             "user:password".
           example: token
+          distribution_scope:
+            description: >
+                The scope of distribution for the index creaded by the request.
+                Will determine what level of protection addition and removal will
+                have.
```

Comment:
```suggestion
                The scope of distribution for the index created by the request.
                This will determine what level of protection the addition will have.
```